defaults-o2.sh

```diff
@@ -53,6 +53,10 @@ overrides:
     tag: "v3.9.4"
     prefer_system_check: |
       which cmake && case `cmake --version | sed -e 's/.* //' | cut -d. -f1,2,3 | head -n1` in [0-2]*|3.[0-8].*|3.9.[0-3]) exit 1 ;; esac
+  pythia:
```

Comment:
So, for my understanding: for O2 builds, in `pythia` you disable `HepMC` in order to avoid conflicts with `HepMC3`. Is this intended? Did you check with @sawenzel if this is OK?

Reply:
(This is just to say that changes are fine for me, approving. Let's just wait for @sawenzel's feedback.)

Reply:
Yes, this is intended. The use of HepMC in pythia is related to standalone event generation where the pythia output is translated in HepMC2 format. I will discuss (or rediscuss in case we did already one month back) with @sawenzel if there is any issue with that.